### PR TITLE
Merge release/20.3 (20.3-rc-2)

### PR DESCRIPTION
**Pull request by ParaskP7**
Merging the release branch after doing a new beta.

The new beta included:
- Fix: https://github.com/WordPress/gutenberg/pull/42517 (via #16925)
- Latest `strings.xml` for WordPress (d1dbc88418431cbd970c4162e129ff616e8d5c4e)
-  ℹ️ New `strings.xml` files for Jetpack, including new `available_languages.xml` file (f281a2cbd365752cad1387001e42c78c9bbf7725 + 5593a9152be83339ea5b10596464a00578abc611)
- Bump version in `version.properties` (2d235a001f935ce014f99cd2970da152f2595ae2)

-----

Merge Conflicts:

```
<<<<<<< merge/release-20.3
    gutenbergMobileVersion = 'v1.79.1'
=======
    gutenbergMobileVersion = 'v1.80.0-alpha3'
>>>>>>> trunk
```

I kept `gutenbergMobileVersion = 'v1.80.0-alpha3'`, which was in `trunk`. PS: @fluiddot or @dcalhoun or @derekblank can you please confirm if the conflict resolution is good and that the `v1.80.0-alpha3` already has the fix? 🙏

### CODE REVIEW

**Comment by ParaskP7** [comment]
PS: The [instrumented-test](https://buildkite.com/automattic/wordpress-android/builds/5638#01822085-6d1c-4bd4-95ea-770713b644d3) check is failing due to this [wPScreenshotTest](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.b3d1e88d8b09c8be/matrices/4975997203780289941/executions/bs.bb960d9fb1c1af5c/test-cases) test failing (maybe its flaky). 🤔 Cc @jkmassel

**Comment by dcalhoun** [comment]
> I kept `gutenbergMobileVersion = 'v1.80.0-alpha3'`, which was in `trunk`. PS: @fluiddot or @dcalhoun or @derekblank can you please confirm if the conflict resolution is good and that the `v1.80.0-alpha3` already has the fix? 🙏

👋🏻 @ParaskP7. Thanks for the ping. 

**TL;DR:** `v1.80.0-alpha3` _does not_ contain the fix, but I still believe we should select `v1.80.0-alpha3`. Not selecting the alpha release would likely break local, daily development. Additionally, the next scheduled release (`1.80.0`) has [since merged](https://github.com/wordpress-mobile/WordPress-Android/pull/16931#event-7038371631), so WordPress-Android `trunk` now contains the bug fix. 

We have run into this [same situation before](https://github.com/wordpress-mobile/WordPress-iOS/pull/17318#issuecomment-943399868). Alpha Gutenberg releases represent code changes upon which Gutenberg's daily development depends. We do not make them on specific cadence, but as needed. Removing one (i.e. rolling back to a lower release) would likely cause errors in daily local development. While the alpha release will not contain the latest crash fix, I do not think that is a major concern. The fix will always arrive when we integrate the next scheduled release.

**Comment by ParaskP7** [comment]
> TL;DR: v1.80.0-alpha3 does not contain the fix, but I still believe we should select v1.80.0-alpha3. Not selecting the alpha release would likely break local, daily development. Additionally, the next scheduled release (1.80.0) has https://github.com/wordpress-mobile/WordPress-Android/pull/16931#event-7038371631, so WordPress-Android trunk now contains the bug fix.

Great, thanks for the confirmation on selecting `v1.80.0-alpha3` @dcalhoun , your clarification makes total sense! 🥇 🙇 👍

**Comment by fluiddot** [comment]
Hey @ParaskP7, thanks for sharing this and merging the release 🙇 . As @dcalhoun mentioned, the version for solving the conflict should be `v1.80.0-alpha3`.

> TL;DR: v1.80.0-alpha3 does not contain the fix, but I still believe we should select v1.80.0-alpha3. Not selecting the alpha release would likely break local, daily development. Additionally, the next scheduled release (1.80.0) has https://github.com/wordpress-mobile/WordPress-Android/pull/16931#event-7038371631, so WordPress-Android trunk now contains the bug fix.

I was checking the commit that was tagged with the alpha version (i.e. `v1.80.0-alpha3`) and noticed that it was created after the fix was merged, so looks like in this case we'd have all changes, including the bug fix, by using the alpha tag 🎊.


Screenshot 2022-07-21 at 19 23 03

**Review by mokagio — APPROVED** [review]
- [x] `versionName` `rc` number and `versionCode` bumps in `build.gradle``version.properties`
- [x] `string.xml` updates

Good to go from a release automation point of view, and from what I make of the comments, also for the conflict resolution

**Comment by ParaskP7** [comment]
> Hey @ParaskP7, thanks for sharing this and merging the release 🙇 . As @dcalhoun mentioned, the version for solving the conflict should be v1.80.0-alpha3.

Thanks for the double confirmation @fluiddot ! 🙇 

> I was checking the commit that was tagged with the alpha version (i.e. v1.80.0-alpha3) and noticed that it was created after the fix was merged, so looks like in this case we'd have all changes, including the bug fix, by using the alpha tag 🎊.

Oh nice, this is good! 🎉

**Comment by ParaskP7** [comment]
Thanks for the review and 👍 @mokagio ! 🙇 ❤️ 🚀

**Comment by ParaskP7** [comment]
👋 @jkmassel !

> PS: The [instrumented-test](https://buildkite.com/automattic/wordpress-android/builds/5638#01822085-6d1c-4bd4-95ea-770713b644d3) check is failing due to this [wPScreenshotTest](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.b3d1e88d8b09c8be/matrices/4975997203780289941/executions/bs.bb960d9fb1c1af5c/test-cases) test failing (maybe its flaky). 🤔 Cc @jkmassel

The `instrumented-test` check is still failing due to `wPScreenshotTest`. Can/Should we disable this check and/or merge this anyway? 🤔

**UPDATE**: I ended-up merging it to `trunk` anyway as I wanted to progress with the `final` release of `20.3` without having any tasks being pending from the previous `beta` release of `20.3-rc-2`. I hope that's okay... 😊 

PS: I also updated this PR with latest `trunk`. This was my last try... ([Buildkite](https://buildkite.com/automattic/wordpress-android/builds/5666#01822507-7c49-40ec-b611-f55f83f6804e) + [Firebase Test Lab](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.b3d1e88d8b09c8be/matrices/6992158711260008367/executions/bs.604f0cd5df3bafb4/test-cases))
